src/mixins/updateIconColor.js

```diff
@@ -6,26 +6,47 @@ export default {
       nodeIcon: null,
     };
   },
+  computed: {
+    icon: {
+      cache: false,
+      get() {
+        if (this.node.definition.get('customIcon')) {
+          const customIcon = this.node.definition.get('customIcon');
+          return atob(customIcon);
+        }
+        return this.nodeIcon;
+      },
```

Comment:
We may not be to set `cache: false` if we switch from `.get` to referencing the reactive property directly. Vue might not be aware of changes to the node definition properties if they are referenced through that `get` method.